### ISSUE TITLE
Charts are not responsive

### ISSUE THREAD

**Issue by wobsoriano** [wobsoriano/coderstats#7]
When you resize the browser to mobile size and put it back to desktop size, the charts doesn't look good.

**Comment by geeks5551** [comment]
hey can i add required responsive ness to the website using advanced css like mediaquery,flexbox,animation,transitions, translation,box-sizing etc  ? looks really  interesting

**Comment by ameybh** [comment]
Hi @wobsoriano. I think this isn't actually an issue. Once you put it to mobile view, if you reload the page, it gets back to the correct size.
Try reloading the page when it's not responsive.

**Comment by wobsoriano** [comment]
> Hi @wobsoriano. I think this isn't actually an issue. Once you put it to mobile view, if you reload the page, it gets back to the correct size.
> Try reloading the page when it's not responsive.

Yeah reloading fixes it too. Just a nice to have

**Comment by ameybh** [comment]
Yes. Although, you should check if the issue persists when mobile devices change orientation.

**Comment by ameybh** [comment]
>Yes. Although, you should check if the issue persists when mobile devices change orientation.

@wobsoriano 
Just checked this and charts take up the whole width on mobile landscape orientation. Probably better to resize them.
